plugin/screenshot/screenshot.py

```diff
@@ -52,20 +52,23 @@ def screenshot_settings():
     def screenshot_clipboard(screen_number: Optional[int] = None):
         """Takes a screenshot of the entire screen and saves it to the clipboard.
         Optional screen number can be given to use screen other than main."""
-        screen = get_screen(screen_number)
-        clipboard_rect(screen.rect)
+        selected_screen = get_screen(screen_number)
+        clipboard_rect(selected_screen.rect)
 
     def screenshot_window_clipboard():
         """Takes a screenshot of the active window and saves it to the clipboard"""
         win = ui.active_window()
         clipboard_rect(win.rect)
 
-
-def screenshot_rect(rect: ui.Rect, title: str = ""):
-    flash_rect(rect)
-    img = screen.capture_rect(rect)
-    path = get_screenshot_path(title)
-    img.write_file(path)
+    def screenshot_rect(
+        rect: ui.Rect, screen_num: Optional[int] = None, title: str = ""
```

Comment:
Should `screen_num` and `title` be swapped around? If for some reason someone wanted to call this from TalonScript it wouldn't be possible to pass `title` without passing a non-None `screen_num`.

Reply:
This seems like a worthwhile change. Otherwise this all tests out OK for me now.

Reply:
Sure I'll change that